packages/suite/src/hooks/suite/useTranslation.ts

```diff
@@ -16,8 +16,9 @@ export const useTranslation = () => {
 
     const translationString = useCallback<TranslationFunction>(
         (id, values) => {
-            if (id && messages[id]) {
-                return intl.formatMessage(messages[id], values);
+            const message = Object.values(messages).find(message => message.id === id);
```

Comment:
`find` is less performant than the key lookup but `useTranslation` is not used too often.